index.html

```diff
@@ -0,0 +1,139 @@
+          alt="Logo"
+          class="w-[17rem] h-[6rem] sm:w-40 sm:h-14 md:w-[17rem] md:h-[6rem]"
+        />
+      </div>
+
+      <div
+        class="border rounded-full sm:w-full w-[45rem] flex justify-center items-center py-1 sm:py-1 sm:px-2 px-2 border-gray-500 md:w-[40rem]"
+      >
+        <svg
+          focusable="false"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          class="w-8 h-10 fill-white"
+          fill="currentColor"
+        >
+          <path
+            d="M15.5 14h-.79l-.28-.27A6.471 6.471 0 0 0 16 9.5 6.5 6.5 0 1 0 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
+          ></path>
+        </svg>
+        <input
+          type="text"
+          class="bg-transparent w-full outline-none text-white px-6 sm:w-full"
+        />
+        <svg
+          class="w-5 h-5"
+          focusable="false"
+          viewBox="0 0 24 24"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill="#4285f4"
+            d="m12 15c1.66 0 3-1.31 3-2.97v-7.02c0-1.66-1.34-3.01-3-3.01s-3 1.34-3 3.01v7.02c0 1.66 1.34 2.97 3 2.97z"
+          ></path>
+          <path fill="#34a853" d="m11 18.08h2v3.92h-2z"></path>
+          <path
+            fill="#fbbc04"
+            d="m7.05 16.87c-1.27-1.33-2.05-2.83-2.05-4.87h2c0 1.45 0.56 2.42 1.47 3.38v0.32l-1.15 1.18z"
+          ></path>
+          <path
+            fill="#ea4335"
+            d="m12 16.93a4.97 5.25 0 0 1 -3.54 -1.55l-1.41 1.49c1.26 1.34 3.02 2.13 4.95 2.13 3.87 0 6.99-2.92 6.99-7h-1.99c0 2.92-2.24 4.93-5 4.93z"
+          ></path>
+        </svg>
+        <img src="./image/cam.svg" alt="" class="w-5 h-5 ml-3" />
+      </div>
+      <div class="text-slate-300 flex gap-2 justify-center items-center">
+        <button class="bg-white/20 py-2 px-4 rounded-sm text-sm">
+          Google search
+        </button>
+
+        <button class="bg-white/20 py-2 px-4 rounded-sm text-sm">
+          I'm feeling lucky
+        </button>
+      </div>
+
+      <p class="text-slate-300 text-sm space-x-2">
+        Google offered in:
+        <span
+          class="text-[#8ab4f8] hover:underline hover:decoration-[#8ab4f8] cursor-pointer"
+          >French</span
+        >
+        <span
+          class="text-[#8ab4f8] hover:underline hover:decoration-[#8ab4f8] cursor-pointer"
+          >English</span
+        >
+        <span
+          class="text-[#8ab4f8] hover:underline hover:decoration-[#8ab4f8] cursor-pointer"
+          >Ikinyarwanda</span
+        >
+      </p>
+    </div>
+
+    <footer class="text-slate-300 flex flex-col bg-[#171717] font-[1rem]">
+      <div
+        class="border-b border-gray-600 text-[15px] px-[30px] py-[15px] text-sm"
+      >
+        <p>Rwanda</p>
+      </div>
+      <div
+        class="flex justify-between w-full px-[30px] py-[15px] text-sm sm:flex-col"
```

Comment:
Same here. Arbitrary values should be defined in the configuration file unless it's a one-time use value